recipes/sage/meta.yaml

```diff
@@ -0,0 +1,27 @@
+
+package:
+  name: sage
+  version: {"0.7.1"}
```

Comment:
```suggestion
{% set name = "sage" %}
{% set version = "0.7.1" %}

package:
  name: {{ name|lower  }}
  version: {{ version  }}
```

Reply:
Also add an
`sha: `
field. You can compute it after downloading the tar file with `sha1sum` for example.